src/dotnet/Program.cs

```diff
@@ -121,22 +121,47 @@ private static int ProcessArgs(string[] args)
                 ["test"] = TestCommand.Run
             };
 
+            int exitCode = 100;
```

Comment:
Is the initialization even necessary?  It looks like all code paths set the variable.